stdlib/public/core/Int128.swift

```diff
@@ -297,8 +297,12 @@ extension Int128 {
   public func dividedReportingOverflow(
     by other: Self
   ) -> (partialValue: Self, overflow: Bool) {
-    _precondition(other != .zero, "Division by zero")
-    if self == .min && other == -1 { return (.min, true) }
+    if _slowPath(other == .zero) {
```

Comment:
Was there an observed performance issue that these `_slowPath` annotations are intended to address, or were they added speculatively?

Reply:
More of a matter of my own curiosity. I'm ok with leaving them in, though they shouldn't matter much.